### PR TITLE
v32.3.0

### PR DESCRIPTION
**Pull request by ArcturusZhang**
This pull request releases minor version v32.3.0

- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] The PR targets the `latest` branch.
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] Apache v2 license headers are included in each file.
 
[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/master/CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # CHANGELOG
 
+## `v32.3.0`
+
+### Updated Packages
+
+| Package Name | API Version |
+| -----------: | :---------: |
```
Comment (jhendrixMSFT) [inline CHANGELOG.md]: Empty table.

Comment (ArcturusZhang) [reply CHANGELOG.md]: This empty table is generated by the tool. Do you mean this empty table should be removed?

Comment (jhendrixMSFT) [reply CHANGELOG.md]: Yeah, looks like it's a bug in the tool.

Comment (ArcturusZhang) [reply CHANGELOG.md]: Got it. I will take some investigation this in the tool later

**Pull request merged**